util.go

```diff
@@ -0,0 +1,49 @@
+package fiberpaginate
+
+import "strings"
+
+/* This file contains all the utility and helper functions that can be reused in paginate */
+
+// isAllowedSort returns true if the given sort string is allowed in the given list of allowedSorts,
+// false otherwise.
+func isAllowedSort(sort string, allowedSorts []string) bool {
```

Comment:
is this redundant if the stdlib has [`slices.Contains`](https://pkg.go.dev/slices#Contains)